Sources/NetworkProtection/ExtensionMessage/ExtensionRequest.swift

```diff
@@ -23,6 +23,7 @@ public enum VPNCommand: Codable {
     case removeSystemExtension
     case removeVPNConfiguration
     case sendTestNotification
+    case simulateKnownFailure(String, Int)
```

Comment:
Do you think it makes sense to provide an NSError here and have anyone interested in this just pull the domain and code themselves? Mainly I'd like to avoid us redefining the domain and codes ourselves and opening up the possibility of them getting out of sync with our own error types.

Reply:
Eh, after thinking about it I don't think it's worth changing, especially if this is just a debug utility.